### ISSUE TITLE
screanreader not announce,whether some contact add to me in contact list too

### ISSUE THREAD

**Issue by AlexandrKozlovskiy** [UnigramDev/Unigram#2340]
When we in contact list,screenreader is not announce,whether the contact add to me in contact list too. As i hear,sighted people can know it,and blindgram (android telelegram client for blind) announce it too for each contact in contact list. So it would be nice,if in unigram screenreder announce this information too.

**Comment by amirmahdifard0** [comment]
@AlexandrKozlovskiy I am one of the most influential people to report accessibility problems and I know well how to fix a problem that does not change for normal users. If possible, I would also report this problem. But once this problem is resolved, there will be a problem for normal users. You should not compare unigram with blindgram, normal users have no priority for blindgram, and blindgram is the only priority for blind users, and blindgram has created this feature for blind users. But unigram can not do that because then it will be a problem for normal users. In unigram, I always make sure that it does not have the slightest effect on the use of normal users before reporting accessibility issues and asking the developer to fix the problem, and then take action.

**Comment by amirmahdifard0** [comment]
@amirmahdifard0 I myself report this problem in the right way at the right opportunity and after obtaining the necessary information.

**Comment by AlexandrKozlovskiy** [comment]
@amirmahdifard0 This can be done,for example using accessibilityName property in view if @FrayxRulez use windowforms framework or another properties,which exists for other frameworks,so it will be no problems for sighted users. I compare blindgram,to pay attention on this problem not only unigram developer,but developers of official telegram client for android,which sometimes forget about accessibility,so blind people should,for example,buy blindgram for android,which have improved accessibility,than official telegram client for android.

**Comment by MarcAnt01** [comment]
> @amirmahdifard0 This can be done,for example using accessibilityName property in view if @FrayxRulez use windowforms framework or another properties,which exists for other frameworks,so it will be no problems for sighted users. I compare blindgram,to pay attention on this problem not only unigram developer,but developers of official telegram client for android,which sometimes forget about accessibility,so blind people should,for example,buy blindgram for android,which have improved accessibility,than official telegram client for android.

Unigram is built using UWP, not WindowsForms

**Comment by FrayxRulez** [comment]
There's nothing visible to users to know if their contacts are mutual. You might get some hint from their profile (you shouldn't have "Share my contact" option) but that's it.
